_posts/2024-08-20-monero-0.18.3.4-released.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "Monero 0.18.3.4 'Fluorine Fermi' released"
+summary: Release removing support for locked transfers
+tags: [releases]
+author: selsta
+image: /blog/assets/2022-07-19-monero-0.18.0.0-released/fluorine-fermi.png
+---
+
+<div align="center">
+  <img src="{{ page.image }}" width="350px">
+</div>
+
+# Overview
+
+This is the v0.18.3.4 release of the Monero software. This release removes support for locked transfers.
+
+Some highlights of this release are:
+
+- Remove support for locked transfers (#[9311](https://github.com/monero-project/monero/pull/9311))
+- Daemon: skip privacy networks that don't have outgoing connections (#[9267](https://github.com/monero-project/monero/pull/9267))
+- Daemon: prevent duplicate txs in fluff queue, fix unintended disconnections (#[9355](https://github.com/monero-project/monero/pull/9355))
+- Daemon: ZMQ DaemonInfo bug fixes (#[9385](https://github.com/monero-project/monero/pull/9385))
+- Wallet: fix stagenet wallet restore height estimate (#[9309](https://github.com/monero-project/monero/pull/9309))
+- Wallet: add Ledger Flex support (#[9430](https://github.com/monero-project/monero/pull/9430))
+- Fix a bug with log rotation mechanism (#[9396](https://github.com/monero-project/monero/pull/9396))
+
+The complete list of changes is [available on GitHub](https://github.com/monero-project/monero/compare/v0.18.3.3...v0.18.3.4), along with [the source code](https://github.com/monero-project/monero/tree/v0.18.3.4).
+
+# Contributors for this Release
+
+This release was the direct result of 8 people who worked, largely unpaid and altruistically, to put out 39 commits containing 535 new lines of code. We'd like to thank them very much for their time and effort. In no particular order they are:
```

Comment:
note that everyone in the contributors list (aside from luigi1111 has an open CCS and is funded)